openrouteservice/src/main/java/heigit/ors/accessibility/AccessibilityAnalyzer.java

```diff
@@ -17,7 +17,8 @@
  *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
- */
+ *//*

```

Comment:
Shouldn't we just delete the whole file rather than commenting out the contents?

Reply:
Might be implemented in the future